avsomr/avsimport.cpp

```diff
@@ -0,0 +1,155 @@
+
+#include <QFileInfo>
```

Comment:
Probably should be in all.h (read: get added there, it is not in yet)

Reply:
added QFile and QFileInfo to all.h